doc/lib_exec.md

```diff
@@ -75,6 +82,40 @@ example:
   does exactly the same as:
   `print foo(42, true, "hello").`
 
+### get_suffix
+
+args:
+  * structure - the structure to get the suffix of
+  * suffix - the suffix to get
+  * parameter_list - if a suffix is a function then this is the ordered list of parameters to be passed to the suffix 0th parameter maps to the 0th item in the list, will be ignored if not of type list, defaulted to `FALSE`
```

Comment:
```suggestion
  * suffix (string) - the suffix to get
  * parameter_list (list) - if a suffix is a function then this is the ordered list of parameters to be passed to the suffix 0th parameter maps to the 0th item in the list, will be ignored if not of type list, defaulted to `FALSE`
```

Reply:
should be fixed now I used the type information style used in the other parts of the documentation